packages/Panel/src/Panel.js

```diff
@@ -25,23 +25,32 @@ export default function Panel(props) {
     a11yText,
     getPushContentElement,
     groupOffsetY,
+    height,
     isCompact,
-    isOpen,
     isInline,
-    isSlideFromLeft,
+    isOpen,
     kind,
-    offsetY,
+    offset,
     onAfterClose,
     onAfterOpen,
     onClose,
+    slideFrom,
     width,
     zIndex,
     ...moreProps
   } = props;
 
+  const offsetTop = "top" in offset ? offset.top : 0;
+
   // Hooks
   const [isVisible, setIsVisible] = React.useState(isOpen);
-  const offsetScroll = useOffsetScroll(offsetY);
+  const offsetScroll = useOffsetScroll(offsetTop);
+
+  const _offset = {
```

Comment:
`currentOffset`? `calculatedOffset`? 
What would best distinguish this value from the prop value?

Reply:
The underscore?  😄